warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py:12: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="../config", config_name="config")
Could not override 'run'. No match in the defaults list.
To append to your default list use +run=proposed-ResNet-18

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py:12: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="../config", config_name="config")
Could not override 'run'. No match in the defaults list.
To append to your default list use +run=proposed-ResNet-18

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py:12: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="../config", config_name="config")
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/hydra/_internal/hydra.py:119: UserWarning: Future Hydra versions will no longer change working directory at job runtime by default.
See https://hydra.cc/docs/1.2/upgrades/1.1_to_1.2/changes_to_job_working_dir/ for more information.
  ret = run_job(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/bin/python3: Error while finding module specification for 'src.train' (ModuleNotFoundError: No module named 'src')
Error executing job with overrides: ['run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']
Traceback (most recent call last):
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 22, in main
    subprocess.run(cmd, check=True)
  File "/home/toma/.local/share/uv/python/cpython-3.11.13-linux-x86_64-gnu/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/bin/python3', '-u', '-m', 'src.train', 'wandb.mode=disabled', 'run=default-run-id', 'results_dir=.research/iteration1', 'trial_mode=true']' returned non-zero exit status 1.

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py:14: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="../config", config_name="config")
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/hydra/_internal/hydra.py:119: UserWarning: Future Hydra versions will no longer change working directory at job runtime by default.
See https://hydra.cc/docs/1.2/upgrades/1.1_to_1.2/changes_to_job_working_dir/ for more information.
  ret = run_job(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'repr' attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. 'repr' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'frozen' attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. 'frozen' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/train.py:149: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="../config", config_name="config")
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/hydra/_internal/hydra.py:119: UserWarning: Future Hydra versions will no longer change working directory at job runtime by default.
See https://hydra.cc/docs/1.2/upgrades/1.1_to_1.2/changes_to_job_working_dir/ for more information.
  ret = run_job(
Error executing job with overrides: ['wandb.mode=disabled', 'run=default-run-id', 'results_dir=.research/iteration1', 'trial_mode=true']
Traceback (most recent call last):
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/train.py", line 160, in main
    raise FileNotFoundError(f"No config file found for run: {run_name}")
FileNotFoundError: No config file found for run: default-run-id

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
Error executing job with overrides: ['run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']
Traceback (most recent call last):
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 28, in main
    subprocess.run(cmd, check=True, cwd=original_cwd)
  File "/home/toma/.local/share/uv/python/cpython-3.11.13-linux-x86_64-gnu/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/bin/python3', '-u', '-m', 'src.train', 'wandb.mode=disabled', 'run=default-run-id', 'results_dir=.research/iteration1', 'trial_mode=true']' returned non-zero exit status 1.

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py:14: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="../config", config_name="config")
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/hydra/_internal/hydra.py:119: UserWarning: Future Hydra versions will no longer change working directory at job runtime by default.
See https://hydra.cc/docs/1.2/upgrades/1.1_to_1.2/changes_to_job_working_dir/ for more information.
  ret = run_job(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'repr' attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. 'repr' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'frozen' attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. 'frozen' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/train.py:149: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="../config", config_name="config")
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/hydra/_internal/hydra.py:119: UserWarning: Future Hydra versions will no longer change working directory at job runtime by default.
See https://hydra.cc/docs/1.2/upgrades/1.1_to_1.2/changes_to_job_working_dir/ for more information.
  ret = run_job(
Error executing job with overrides: ['run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']
Traceback (most recent call last):
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/train.py", line 160, in main
    raise FileNotFoundError(f"No config file found for run: {run_name}")
FileNotFoundError: No config file found for run: default-run-id

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
Error executing job with overrides: ['run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']
Traceback (most recent call last):
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 26, in main
    subprocess.run(cmd, check=True, cwd=original_cwd)
  File "/home/toma/.local/share/uv/python/cpython-3.11.13-linux-x86_64-gnu/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/bin/python3', '-u', '-m', 'src.train', 'run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']' returned non-zero exit status 1.

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py:14: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="../config", config_name="config")
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/hydra/_internal/hydra.py:119: UserWarning: Future Hydra versions will no longer change working directory at job runtime by default.
See https://hydra.cc/docs/1.2/upgrades/1.1_to_1.2/changes_to_job_working_dir/ for more information.
  ret = run_job(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'repr' attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. 'repr' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'frozen' attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. 'frozen' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/train.py:149: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="../config", config_name="config")
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/hydra/_internal/hydra.py:119: UserWarning: Future Hydra versions will no longer change working directory at job runtime by default.
See https://hydra.cc/docs/1.2/upgrades/1.1_to_1.2/changes_to_job_working_dir/ for more information.
  ret = run_job(
Error executing job with overrides: ['run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']
Traceback (most recent call last):
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/train.py", line 164, in main
    raise FileNotFoundError(f"No config file found for run: {run_name}")
FileNotFoundError: No config file found for run: default-run-id

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
Error executing job with overrides: ['run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']
Traceback (most recent call last):
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 26, in main
    subprocess.run(cmd, check=True, cwd=original_cwd)
  File "/home/toma/.local/share/uv/python/cpython-3.11.13-linux-x86_64-gnu/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/bin/python3', '-u', '-m', 'src.train', 'run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']' returned non-zero exit status 1.

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py:14: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="../config", config_name="config")
Could not override 'run'. No match in the defaults list.
To append to your default list use +run=proposed-ResNet-18

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'repr' attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. 'repr' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'frozen' attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. 'frozen' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
Could not override 'run'. No match in the defaults list.
To append to your default list use +run=proposed-ResNet-18

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 18, in <module>
    main()
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 15, in main
    subprocess.run(cmd, check=True)
  File "/home/toma/.local/share/uv/python/cpython-3.11.13-linux-x86_64-gnu/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/bin/python3', '-u', '-m', 'src.train', 'run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']' returned non-zero exit status 1.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'repr' attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. 'repr' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'frozen' attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. 'frozen' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
Error executing job with overrides: ['run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']
Traceback (most recent call last):
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/train.py", line 173, in main
    cfg: DictConfig = OmegaConf.merge(cfg_cli, cfg_run)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
omegaconf.errors.ConfigKeyError: Key 'run_id' is not in struct
    full_key: run_id
    object_type=dict

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 18, in <module>
    main()
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 15, in main
    subprocess.run(cmd, check=True)
  File "/home/toma/.local/share/uv/python/cpython-3.11.13-linux-x86_64-gnu/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/bin/python3', '-u', '-m', 'src.train', 'run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']' returned non-zero exit status 1.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'repr' attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. 'repr' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'frozen' attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. 'frozen' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
Error executing job with overrides: ['run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']
Traceback (most recent call last):
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/train.py", line 181, in main
    cfg.trial_limit_batches = 2
    ^^^^^^^^^^^^^^^^^^^^^^^
omegaconf.errors.ConfigAttributeError: Key 'trial_limit_batches' is not in struct
    full_key: trial_limit_batches
    object_type=dict

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 18, in <module>
    main()
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 15, in main
    subprocess.run(cmd, check=True)
  File "/home/toma/.local/share/uv/python/cpython-3.11.13-linux-x86_64-gnu/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/bin/python3', '-u', '-m', 'src.train', 'run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']' returned non-zero exit status 1.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'repr' attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. 'repr' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'frozen' attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. 'frozen' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/preprocess.py:73: UserWarning: RobustBench not found – falling back to clean CIFAR-10 test set.
  warnings.warn("RobustBench not found – falling back to clean CIFAR-10 test set.")
  0%|          | 0.00/170M [00:00<?, ?B/s]  0%|          | 32.8k/170M [00:00<14:28, 196kB/s]  0%|          | 65.5k/170M [00:00<14:14, 199kB/s]  0%|          | 98.3k/170M [00:00<14:13, 200kB/s]  0%|          | 229k/170M [00:00<06:34, 432kB/s]   0%|          | 459k/170M [00:00<03:40, 770kB/s]  1%|          | 918k/170M [00:00<01:57, 1.45MB/s]  1%|          | 1.84M/170M [00:01<01:00, 2.77MB/s]  2%|▏         | 3.70M/170M [00:01<00:30, 5.44MB/s]  4%|▍         | 6.82M/170M [00:01<00:17, 9.53MB/s]  6%|▌         | 9.96M/170M [00:01<00:12, 12.4MB/s]  8%|▊         | 13.1M/170M [00:01<00:11, 14.3MB/s] 10%|▉         | 16.2M/170M [00:02<00:09, 15.6MB/s] 11%|█▏        | 19.3M/170M [00:02<00:09, 16.6MB/s] 13%|█▎        | 22.5M/170M [00:02<00:08, 17.1MB/s] 15%|█▌        | 25.6M/170M [00:02<00:08, 17.4MB/s] 17%|█▋        | 28.7M/170M [00:02<00:07, 18.0MB/s] 19%|█▊        | 31.8M/170M [00:02<00:06, 20.6MB/s] 20%|█▉        | 34.0M/170M [00:02<00:06, 19.6MB/s] 21%|██        | 36.0M/170M [00:03<00:07, 17.4MB/s] 22%|██▏       | 38.0M/170M [00:03<00:07, 17.2MB/s] 24%|██▍       | 41.2M/170M [00:03<00:07, 18.0MB/s] 26%|██▌       | 43.6M/170M [00:03<00:06, 19.4MB/s] 27%|██▋       | 45.7M/170M [00:03<00:06, 18.1MB/s] 28%|██▊       | 47.5M/170M [00:03<00:07, 17.5MB/s] 29%|██▉       | 50.1M/170M [00:03<00:06, 19.4MB/s] 31%|███       | 52.1M/170M [00:03<00:06, 18.4MB/s] 32%|███▏      | 54.0M/170M [00:04<00:06, 17.3MB/s] 33%|███▎      | 56.6M/170M [00:04<00:06, 17.8MB/s] 35%|███▍      | 59.0M/170M [00:04<00:06, 18.5MB/s] 36%|███▌      | 60.9M/170M [00:04<00:06, 17.6MB/s] 37%|███▋      | 62.9M/170M [00:04<00:05, 18.3MB/s] 38%|███▊      | 65.1M/170M [00:04<00:05, 18.5MB/s] 39%|███▉      | 66.9M/170M [00:04<00:05, 17.3MB/s] 41%|████      | 69.3M/170M [00:04<00:05, 19.0MB/s] 42%|████▏     | 71.3M/170M [00:04<00:05, 18.9MB/s] 43%|████▎     | 73.2M/170M [00:05<00:05, 17.3MB/s] 44%|████▍     | 75.7M/170M [00:05<00:04, 19.2MB/s] 46%|████▌     | 77.7M/170M [00:05<00:05, 18.4MB/s] 47%|████▋     | 79.6M/170M [00:05<00:05, 17.4MB/s] 48%|████▊     | 82.0M/170M [00:05<00:04, 19.2MB/s] 49%|████▉     | 84.0M/170M [00:05<00:04, 18.3MB/s] 50%|█████     | 85.9M/170M [00:05<00:04, 17.2MB/s] 52%|█████▏    | 88.5M/170M [00:05<00:04, 18.0MB/s] 53%|█████▎    | 91.0M/170M [00:06<00:04, 19.6MB/s] 55%|█████▍    | 93.0M/170M [00:06<00:04, 18.8MB/s] 56%|█████▌    | 94.9M/170M [00:06<00:04, 16.4MB/s] 57%|█████▋    | 97.8M/170M [00:06<00:04, 17.0MB/s] 59%|█████▉    | 101M/170M [00:06<00:03, 17.6MB/s]  61%|██████    | 104M/170M [00:06<00:03, 17.9MB/s] 63%|██████▎   | 107M/170M [00:06<00:03, 20.7MB/s] 64%|██████▍   | 109M/170M [00:07<00:03, 18.2MB/s] 65%|██████▌   | 111M/170M [00:07<00:03, 17.5MB/s] 67%|██████▋   | 113M/170M [00:07<00:03, 17.7MB/s] 68%|██████▊   | 115M/170M [00:07<00:03, 17.6MB/s] 69%|██████▊   | 117M/170M [00:07<00:03, 16.7MB/s] 70%|███████   | 120M/170M [00:07<00:02, 18.7MB/s] 71%|███████▏  | 122M/170M [00:07<00:02, 17.9MB/s] 72%|███████▏  | 123M/170M [00:07<00:02, 17.0MB/s] 74%|███████▍  | 126M/170M [00:08<00:02, 16.3MB/s] 76%|███████▌  | 129M/170M [00:08<00:02, 16.6MB/s] 77%|███████▋  | 132M/170M [00:08<00:02, 16.9MB/s] 79%|███████▉  | 135M/170M [00:08<00:02, 17.0MB/s] 81%|████████  | 138M/170M [00:08<00:01, 17.1MB/s] 83%|████████▎ | 141M/170M [00:08<00:01, 17.6MB/s] 84%|████████▍ | 144M/170M [00:09<00:01, 17.4MB/s] 86%|████████▌ | 147M/170M [00:09<00:01, 17.5MB/s] 88%|████████▊ | 150M/170M [00:09<00:01, 17.7MB/s] 90%|████████▉ | 153M/170M [00:09<00:01, 17.4MB/s] 92%|█████████▏| 156M/170M [00:09<00:00, 17.6MB/s] 93%|█████████▎| 159M/170M [00:09<00:00, 17.7MB/s] 95%|█████████▌| 162M/170M [00:10<00:00, 17.5MB/s] 97%|█████████▋| 165M/170M [00:10<00:00, 17.8MB/s] 99%|█████████▊| 168M/170M [00:10<00:00, 17.4MB/s]100%|██████████| 170M/170M [00:10<00:00, 16.2MB/s]
Error executing job with overrides: ['run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']
Traceback (most recent call last):
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/train.py", line 234, in main
    stats = run_adaptation(cfg)
            ^^^^^^^^^^^^^^^^^^^
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/train.py", line 54, in run_adaptation
    backbone = create_backbone(cfg).to(device)
               ^^^^^^^^^^^^^^^^^^^^
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/model.py", line 19, in create_backbone
    model = timm.create_model(
            ^^^^^^^^^^^^^^^^^^
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/timm/models/_factory.py", line 117, in create_model
    model = create_fn(
            ^^^^^^^^^^
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/timm/models/resnet.py", line 1328, in resnet18
    return _create_resnet('resnet18', pretrained, **dict(model_args, **kwargs))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/timm/models/resnet.py", line 641, in _create_resnet
    return build_model_with_cfg(ResNet, variant, pretrained, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/timm/models/_builder.py", line 415, in build_model_with_cfg
    model = model_cls(**kwargs)
            ^^^^^^^^^^^^^^^^^^^
TypeError: ResNet.__init__() got an unexpected keyword argument 'cache_dir'

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 18, in <module>
    main()
  File "/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/main.py", line 15, in main
    subprocess.run(cmd, check=True)
  File "/home/toma/.local/share/uv/python/cpython-3.11.13-linux-x86_64-gnu/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/bin/python3', '-u', '-m', 'src.train', 'run=proposed-ResNet-18', 'results_dir=.research/iteration1', 'wandb.mode=disabled', 'trial_mode=true']' returned non-zero exit status 1.
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'repr' attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. 'repr' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'frozen' attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. 'frozen' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/preprocess.py:73: UserWarning: RobustBench not found – falling back to clean CIFAR-10 test set.
  warnings.warn("RobustBench not found – falling back to clean CIFAR-10 test set.")
warning: No `requires-python` value found in the workspace. Defaulting to `>=3.11`.
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'repr' attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. 'repr' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/.venv/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The 'frozen' attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. 'frozen' is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
  warnings.warn(
/home/toma/t-80-8-b-03/_work/airas-20251020-120659-matsuzawa/airas-20251020-120659-matsuzawa/src/preprocess.py:73: UserWarning: RobustBench not found – falling back to clean CIFAR-10 test set.
  warnings.warn("RobustBench not found – falling back to clean CIFAR-10 test set.")
